packages/twenty-server/src/modules/workspace-member/standard-objects/options/workspace-member-locale.options.ts

```diff
@@ -0,0 +1,242 @@
+import { FieldMetadataComplexOption } from 'src/engine/metadata-modules/field-metadata/dtos/options.input';
+
+export enum WorkspaceMemberLocaleEnum {
```

Comment:
Sorry but I don't think we'll realistically want to cover these any day. I thought we could do something simpler, no?

Cf Discord message:
<img width="296" alt="Screenshot 2024-07-16 at 10 56 00" src="https://github.com/user-attachments/assets/340a5170-7681-423a-8727-720bb4de1995">

Reply:
Done!